core/lib/dal/src/blocks_dal.rs

```diff
@@ -1272,12 +1288,20 @@ impl BlocksDal<'_, '_> {
     pub async fn get_ready_for_execute_l1_batches(
         &mut self,
         limit: usize,
+        commited_tx_confirmed: bool,
```

Comment:
Checking if commitBatches is confirmed looks unnecessary here, it already checks that `eth_prove_tx_id is NOT NULL` which should be enough